tests/unit/components/popular-categories/PopularCategories.spec.jsx

```diff
@@ -38,6 +38,7 @@ const title = 'common.popularCategories'
 const description = 'studentHomePage.popularCategories.description'
 
 describe('PopularCategories', () => {
+
   beforeEach(async () => {
     await waitFor(() => {
       mockAxiosClient.onGet(URLs.categories.get).reply(200, mockResponse)
```

Comment:
```suggestion
      mockAxiosClient.onGet(new RegExp(URLs.categories.get)).reply(200, mockResponse)
```
Try to add `RegExp` for URL mapping on GET request. Probably it's because before `baseService` we have to put searchParams inside the `axiosClient` call, but right now we generate the resulting URL with search params with another approach (`getFullUrl`).